src/data/markdown/translated-guides/en/02 Using k6/20 Execution API.md

```diff
@@ -0,0 +1,131 @@
+|---------------------|---------|--------------------------------------------------------------------------|
+| name                | string  | It is the assigned name of the running scenario.                                |
+| executor            | string  | It is the name of the running [Executor](https://k6.io/docs/using-k6/scenarios/#executors).                                                |
+| startTime           | integer | It reperesents the unix timestamp in milliseconds when the scenario started.                                                                           |
+| progress            | float   | Percentage in a 0 to 1 interval of the scenario progress.                    |
+| iterationInInstance | integer | The unique identifier of the single iteration across a single running instance. |
+| iterationInTest     | integer | The unique identifier of the single iteration across the entire test. It is unique both in local, in cloud and in distributed or segmented executions. |
+
+### Instance
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| iterationsInterrupted                | integer  | The amount of iterations interrupted.                                |
+| iterationsCompleted     | integer | The amount of iterations completed.  |
+| vusActive            | integer  | The amount of VUs active.                                                |
+| vusInitialized           | integer | The amount of VUs initialized from the start of the test.                                                                           |
+| currentTestRunDuration            | float   | The time passed from the start of the current test run in milliseconds.                    |
+
+### VU
+
+| Field               | Type    | Description                                                              |
+|---------------------|---------|--------------------------------------------------------------------------|
+| iterationInInstance | integer | The identifier of the iteration in the current instance.                                                                 |
+| iterationInScenario | integer | The identifier of the iteration in the current scenario.                  |
+| idInInstance        | integer | The identifier of the VU across the instance.                            |
+| idInTest            | integer | The global-unique identifier of the VU.                                  |
+
+### Get unqiue data once
+
+One of the common cases,  it's getting an unqiue and incremental index for accessing a dataset's item only once time per test.
+
+The `scenario.iterationInTest` field can be used for this case. Let's show an example:
+
+```
+import exec from "k6/execution";
+import {SharedArray} from "k6/data";
+
+const data = new SharedArray(); // populate it with a real dataset
```

Comment:
I think you should still return a workable SharedArray so this example can be tried by people instead of getting an error